packages/gestalt-datepicker/src/DatePicker.js

```diff
@@ -173,6 +175,13 @@ const DatePickerWithForwardRef: AbstractComponent<Props, HTMLInputElement> = for
 ): Node {
   const innerInputRef = useRef<null | HTMLInputElement>(null);
   useImperativeHandle(ref, () => innerInputRef.current);
+  const { datePickerHandlers } = useGlobalEventsHandler() || {
+    datePickerHandlers: undefined,
+  };
+
+  useEffect(() => {
+    if (datePickerHandlers?.onMount) datePickerHandlers?.onMount();
```

Comment:
This may get called in non-`onMount` situations right, e.g if another handler is changed?

Reply:
should this just be. Not sure if onMount should be changeable 
```
useEffect()=>{
 if (datePickerHandlers?.onMount) datePickerHandlers?.onMount();
},[])
```